keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/BakeHistory.kt

```diff
@@ -0,0 +1,6 @@
+package com.netflix.spinnaker.keel.bakery.artifact
+
+interface BakeHistory {
+  fun contains(appVersion: String, baseAmiVersion: String): Boolean
+  fun add(appVersion: String, baseAmiVersion: String, regions: Collection<String>, taskId: String)
+}
```

Comment:
Simple interface to record and query whether we've baked a particular appversion / base_ami_version combination